ui/packages/consul-partitions/vendor/consul-partitions/services.js

```diff
@@ -0,0 +1,9 @@
+(services => services({
+  "component:consul/partition/selector": {
+    "class": "consul-ui/components/consul/partition/selector"
```

Comment:
Use this class for the `Consul::Partition::Selector` component _only_ if this config file has been 'imported' (which is only when Consul has set the `PartitionsEnabled` feature flag to `true`)